data/reporter.js

```diff
@@ -125,7 +125,6 @@ self.port.on("app_e10s_enabled", function(appE10sEnabled, addonE10sEnabled) {
 self.port.on("have_addon_reports", initialize);
 self.port.on("reset", function() {
     document.initialized = false;
-    initialize(document.addonReports);
```

Comment:
The panel is reloaded as soon as it's being opened the next time. If we `initialize` here, the panel will be initialized with the old data, instead of the new.